src/ostorlab/utils/ip.py

```diff
@@ -0,0 +1,18 @@
+"""Utils to handle IP operations."""
+from typing import Optional
+
+import requests
+
+
+def get_public_ip() -> Optional[str]:
```

Comment:
We must get the IP from the machine, not the public now. Absolutely not make any API call to 3rd party.